system/services/enum/EnumService.cfc

```diff
@@ -45,6 +45,62 @@ component {
 		return itemsWithLabelsAndDescriptions;
 	}
 
+	public struct function getStructItems( required string enum, string value = "" ) {
```

Comment:
I like this idea, could we rename the method, and rejig the signature though:

```cfc
public struct function getEnumProperties( required string enum, array restrictToKeys=[], array properties=[ "label", "description" )
```

The properties can be dynamically used to loop over in the `translateResource()` calls. This way calling code can customize implementations, i.e. add `iconClass`, etc.

Also, note the preferred use of an `array` type, rather than using string lists.